portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,35 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Mario! I am supposed to be a comment...but I'm not quite there yet.");
-  }
-}
+    private HashMap<String, String> comments;
```

Comment:
You can use private Map here instead of HashMap, since you only need to deal the Map interface.

Reply:
Done.